### ISSUE TITLE
start.sh: line 6: screen: command not found

### ISSUE THREAD

**Issue by trentwiles** [tbiens/icarus#3]
Might be a dumb error, I ran the setup script as root and got this error:

start.sh: line 6: screen: command not found

**Comment by ghost** [comment]
You don't have screen installed, `apt install screen` or similar for your OS.

The program is run interactively. Ctrl+A+D to detach.

**Comment by ghost** [comment]
If you're using Red Hat or a flavor of it, it's deprecated. You can just remove the screen command anyways or replace it with tmux.

**Comment by tbiens** [comment]
I'm sorry for not seeing this. I have been working on: https://www.largnet.ca/largblog/largncm an NCM that does security reports as per the network device hardening guides. 

Today I learnt that screen is deprecated. https://access.redhat.com/solutions/4136481

I like both screen and tmux and I'm more than happy to switch over to Tmux. 

I have updated the start.sh to include logic to check for tmux and docker. I just installed Ubuntu 20 lts and will be testing.
